src/de/jost_net/JVerein/gui/action/OpenInsertVariableDialogAction.java

```diff
@@ -44,6 +44,7 @@ public void handleAction(Object context)
       }
       else if (context instanceof Map)
       {
+        //noinspection unchecked
```

Comment:
Damit wird in Eclipse aber immer noch die Warning gelb angezeigt.

Reply:
In IntelliJ tatsächlich nicht

Reply:
Ist das denn eine Intellij Option? In Java gehen die doch immer mit @ los.